guides/Network nodes/fleek-network-how-to-install-a-node-easily-with-the-assisted-installer.md

```diff
@@ -0,0 +1,204 @@
+
+Operating system are highly customisable and some users might much prefer to setup the Network Node required applications and dependencies themselves. Some of these users have a lot of system level knowledge, which not all other users have. By putting down the steps programmatically, or in a sequence of commands, we help reduce the frustration that some users might feel otherwise when following guides or longe descriptive processes.
+
+The assisted installer is one of the many options we provide to onboard users of any shape, geographical location, technical knowledge into the Fleek Network, an open network made for everybody.
+
+## How does it Work?
+
+A user has to copy or type the command provided in the guide or our website and paste it to a terminal connected to the machine or server where the Network Node will be installed and run.
+
+Once the user executes the command, a process is initiated that attempts to provide the user with enough information about what's going on, or what can optionally happen, e.g., since [Git - a distributed version control system](https://git-scm.com/) is a required application and if not found, the user will be informed and a request to install it will be presented as a user shell prompt. The users will be able to accept or deny the request and the installer will only execute accordingly. Most requirements are dependencies, such as, libraries or packages that your system must have for our software to run. Apart from the [Ursa CLI](https://github.com/fleek-network/ursa) we're developing, the dependencies are provided by third-parties which you as a user might be interested in checking up.
+
+After the installations, there'll be a request to provide a valid custom domain name of the user control, to decorate the server public [IP address](https://en.wikipedia.org/wiki/IP_address), where an attempt to [secure it](./fleek-network-securing-a-node-with-ssl-tls) with SSL/TLS Certificates will be handled by leveraging the [Let's Encrypt](https://letsencrypt.org/) API for the matter.
+
+Once the SSL/TLS certification is provided, the Stack provided by [Docker](https://www.docker.com/) (a program that helps accelerate and simplify the distribution of applications via containers) is restarted. The services in the Stack, as described in our guide [](./fleek-network-node-healthchecks#processes) will then start with the final changes made during the certification process (mainly the reverse proxy, that's responsible to map the public port 80 to the internal 4069).
+
+If all goes successfully, the user will have a running node that is secured with SSL/TSL and the operations monitored by our suggested apps Prometheus and Grafana available to you easily in our opinonated Docker Stack.
+
+## Which operating systems are supported?
+
+At the present time, the assisted installer is supporting latest Ubuntu (22.04 LTS) and Debian (version 11), or earlier. Including ArchLinux, that has rolling updates.
```

Comment:
Minor feedback, I would list them for better readability/finding them when reading the guide